components/sections/allocations/SectionAllocationsList.tsx

```diff
@@ -0,0 +1,140 @@
+import	React, {ReactElement}	    from	'react';
+import	{List}						from	'@yearn-finance/web-lib/layouts';
+import	{Card}						from	'@yearn-finance/web-lib/components';
+import	* as utils					from	'@yearn-finance/web-lib/utils';
+import	{Chevron}					from	'@yearn-finance/web-lib/icons';
+import {format} from '@yearn-finance/web-lib/utils';
+
+const	ProtocolBox = React.memo(function ProtocolBox({protocol}: {protocol: any}): ReactElement {
+	function	renderSummary(p: {open: boolean}): ReactElement {
+		return (
+			<div className={`rounded-default relative grid h-20 w-[965px] grid-cols-22 bg-neutral-0 py-4 px-6 transition-colors md:w-full ${p.open ? '' : 'hover:bg-neutral-100'}`}>
+				<div className={'min-w-32 col-span-5 flex flex-row items-center'}>
+					<div className={'text-neutral-500'}>
+						<div className={'flex-row-center'}>
+							<div>
+								<b>{protocol.name}</b>
+							</div>
+						</div>
+					</div>
+				</div>
+				<div className={'min-w-36 cell-end col-span-4 mr-5 flex flex-row items-center tabular-nums'}>
+					<div>
+						<b>{`${utils.format.amount(protocol.tvl, 2)}$`}</b>
+						<p className={'text-sm'}>{`${utils.format.amount(protocol.totalDebtRatio, 2)}%`}</p>
+					</div>
+				</div>
+				<div className={'min-w-36 cell-end col-span-4 mr-5 flex flex-row items-center tabular-nums'}>
+					<div>
+						{protocol.strategiesAmount}
+					</div>
+				</div>
+				<div className={'min-w-36 cell-end col-span-5 flex flex-row items-center tabular-nums'}>
+					<div>
+						{protocol.allocatedStrategies}
+					</div>
+				</div>
+				<div className={'min-w-36 cell-end col-span-4 flex flex-row items-center'}>
+					<div className={'ml-2'}>
+						<Chevron
+							className={`h-6 w-6 text-accent-500 transition-transform ${p.open ? '-rotate-90' : '-rotate-180'}`} />
+					</div>
+				</div>
+			</div>
+		);
+	}
+	function	RenderDetail(): ReactElement {
+		const _strategiesList = Object.keys(protocol.strategiesTVL).map((strategy)=>{
+			return {
+				name: strategy,
+				tvl: protocol.strategiesTVL[strategy]
+			};
+		});
+		const strategiesList = _strategiesList.sort((a, b): number => {
+			return b.tvl - a.tvl;
+		});
+		return (
+			<div className={'my-10'}>
+				{
+					strategiesList.map((strategy): ReactElement => {
+						return (
+							<div className={'mb-10'} key={strategy.name}>
+								<div className={'mx-auto flex w-10/12 flex-col'}>
+									<span className={'mb-2 flex flex-row items-center justify-between'}>
+										<p className={'text-left text-neutral-500'}>{`${strategy.name}`}</p>
+										<b className={'text-left text-accent-500'}>
+											{`${format.amount(strategy.tvl / protocol.tvl * 100)}%`}
```

Comment:
Need to add check that `strategy.tvl` and `protocol.tvl` not a `0`.  Dividing by 0 will show the string "Not a number%" on the page